modules/utils/docker/entrypoint_nats.sh

```diff
@@ -18,20 +18,42 @@ else
       generate_identity_id
   fi
 
+  # Do not continue in case halow init has not finished
+  while ps aux | grep [i]nit_halow > /dev/null; do
+      sleep 1
+  done
+
   echo "set bridge ip"
-  generate_br_lan_ip
+  generate_bridge_ip
```

Comment:
As mentioned above: there will be quite a few bridges, if this is only for br-lan then let's keep it in the name :)

Reply:
Bridge name is in config parameters.

Reply:
How about generate_lan_bridge_ip or something clearer like that? 🙂

Reply:
fixed https://github.com/tiiuae/mesh_com/pull/319/commits/5353cf47e111210414924b59e97faec42b8f29cc